lib/util/toml.ts

```diff
@@ -0,0 +1,7 @@
+import { getStaticTOMLValue, parseTOML } from 'toml-eslint-parser';
+import type { JsonValue } from 'type-fest';
+
+export function parse(input: string): JsonValue {
```

Comment:
```suggestion
export function parse(input: string): unknown {
```
I would prefer unknown like we do for JSON, so calling code is responsible to validate. best to use schema